test-data/users.data.ts

```diff
@@ -0,0 +1,4 @@
+export const userCredentials = {
+    userEmail: "tomek.pawlak60@gmail.com",
+    userPassword: "Password123!"
+}
```

Comment:
good practice it to add empty line at the end of file

please fix all files.

Reply:
Solved.